build.sbt

```diff
@@ -7,7 +7,7 @@ lazy val root = (project in file("."))
   .settings(
     organization := "org.broadinstitute.dsp",
     name := "helm-scala-sdk",
-    version := "0.0.1-RC1",
+    version := "0.0.1-RC3",
```

Comment:
Maybe need to rebase this PR? I had changed the version to RC2, weird it's not showing in this diff

Reply:
Ohh weird it's RC1 in master branch. Did I forget to push? 🤦 

This looks fine then, I think.

Reply:
Yeah :( sorry

```
rtitle@helm-scala-sdk[master]$ git status
On branch master
Your branch is ahead of 'origin/master' by 1 commit.
  (use "git push" to publish your local commits)
nothing to commit, working tree clean
```

Reply:
No worries at all. I just thought perhaps we were bumping versions off repo for the time being or something.